src/Helper/MyParcelCollection.php

```diff
@@ -141,14 +139,6 @@ public function getConsignmentByApiId($id)
         return null;
     }
 
-    /**
-     * @return string
-     */
-    public function getLabelPdf()
```

Comment:
Thanks for your quick feedback! I'll put it back. Could you describe what it can be used for so I can document it? I removed it because it wasn't documented or used in our own products.

Reply:
Sure, we use it to return a response with the label. So similar to you example [here](https://github.com/myparcelnl/sdk#download-labels) but with a custom name through the `Content-disposition` header. 

Primarily though, it is used within a framework (Symfony), which takes care of actually sending the headers - outside the controller logic. It also needs to handle some post-response event logic and close it's kernel. Calling `exit` kills off the process, so the framework can no longer finish it's work